src/library/mixin_helper.lua

```diff
@@ -528,7 +528,7 @@ function mixin_helper.create_custom_window_change_event(...)
 
     local function trigger_func(window, immediate)
         if type(window) == "boolean" and window then
-            for win in event.target_iterator() do
+            for _ in event.target_iterator() do
```

Comment:
This line was correct as it was. The typo is in lines 533 and 535 below which should reference `win` instead of `window`.

Reply:
Thanks, will fix.